tox.ini

```diff
@@ -66,6 +66,14 @@ commands =
     pylint --rcfile=pylintrc --output-format=parseable {toxinidir}/src/pymovements {toxinidir}/tests
 
 
+[testenv:mypy]
+deps =
+    mypy
```

Comment:
I think when you add this line with a single dot, this will install the package dependencies and we're not forced to ignore the missing imports.

```suggestion
    mypy
    .
```

Also see this dependency definition for running pytest as an example with additional extras:
https://github.com/aeye-lab/pymovements/blob/07f13f60b8b0669ccb5eeda903b0b48d9a93843d/tox.ini#L7-L10

In theory we don't need to specify mypy as a dependency this way in the `setup.cfg`. Then it will be only installed for the venv which is used for running mypy checking.
Although I think it would be best to include it here:
https://github.com/aeye-lab/pymovements/blob/07f13f60b8b0669ccb5eeda903b0b48d9a93843d/setup.cfg#L32-L36

It would be very nice if you could also put `flake8` and `pylint` together with `mypy` into the `setup.cfg`, as they seem to be missing from the test-extra dependency.

Also, while you're at it, please remove `pytest-cov` from the definition here, 
https://github.com/aeye-lab/pymovements/blob/07f13f60b8b0669ccb5eeda903b0b48d9a93843d/tox.ini#L7-L10
it is redundant as it's already in the `setup.cfg`.

I think then specification of test dependencies would be a bit more consistent than in the current state.
May be a bit out of the original scope of the PR, though, but the changes are so tiny :man_shrugging: